src/components/AddNewBook.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class AddNewBook extends React.PureComponent {
+  render() {
+    return (
+      <div className="add-book-container">
+        <h2>ADD NEW BOOK</h2>
+        <form>
+          <input type="text" placeholder="Book Title" />
+          <input type="text" placeholder="Author" />
+        </form>
```

Comment:
- [ ] [REQUIRED] Please, make sure that you have a submit button with the label "Add Book" as mentioned in the project requirements.

Reply:
@OmarRamoun Thank you very much, I totally forgot about it. I will start working on it right now!